src/routes/login/components/LoginPage.js

```diff
@@ -82,8 +82,7 @@ module.exports = function LoginPage({ onPressLogin, isAuthenticated }) {
 };
 
 module.exports.propTypes = {
-    onPressLogin: T.func.isRequired,
-    isAuthenticated: T.bool.isRequired
```

Comment:
this prop was throwing a warning because sometimes it is undefined, and found it's not used, so gave it the 🥾